src/components/GoogleMapApi/GoogleMapApi.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { GoogleMap, Marker, useJsApiLoader } from '@react-google-maps/api';
+
+const containerStyle = {
+  width: '100%',
+  height: '500px',
+};
+
+function MyComponent({ center }) {
+  const { isLoaded } = useJsApiLoader({
+    id: 'google-map-script',
+    googleMapsApiKey: 'AIzaSyAUo-MP_Un5CB4x9r2QxLxZSxW5-k8iBGc',
+  });
+
+  const [map, setMap] = React.useState(null);
+
+  const onLoad = React.useCallback(
```

Comment:
```suggestion
  const onLoad = useCallback(
```
지금 구현하신 방법 보다 import 해서 직접 가져오는 방식이 가독성 측면에서 더 좋은 방식입니다.
수정해주세요